vspace/src/main/java/edu/asu/diging/vspace/web/staff/api/FileApiController.java

```diff
@@ -0,0 +1,67 @@
+
+package edu.asu.diging.vspace.web.staff.api;
```

Comment:
why are the file controllers in the `api` package? unless they are returning json, they should not be in here.